test/views/declaration/AdditionalInformationViewSpec.scala

```diff
@@ -143,7 +143,7 @@ class AdditionalInformationViewSpec extends UnitViewSpec with AdditionalInformat
       view.select("table>tbody>tr>th:nth-child(1)").text() mustBe "12345-12345"
 
       val removeButton = view.select("table>tbody>tr>th:nth-child(2)>button")
-      removeButton.text() mustBe messages(removeCaption)
+      removeButton.text() mustBe "site.remove site.remove.hint"
```

Comment:
I got tricked by this, thinking it is single message. Could you split it into 2 lines and use for e.g. `include("...")`?

Reply:
Could do.  But isn't it better to be more precise?  (I.e. use mustBe).   To be honest, I'm not 100% convinced of the value of tests like this.  I just fixed what I broke.

Reply:
If we think it's worth testing the hint text separately then maybe we need to extract the text of the two spans contained within the button and test them separately?